### DISCUSSION TITLE
Failed to find a valid digest in the 'integrity' attribute for resource

### DISCUSSION THREAD

**Discussion by ilfungo** [kimai/kimai#4306] [On-Premise: Installation & Updates] [answered]
I get this error "Failed to find a valid digest in the 'integrity' attribute for resource 'http://kimai.test/build/app.9acfcfc8.js' with computed SHA-384 integrity 'ghF9wKKz/PkI1XqAfMwh6PYqUZ1FhhBdzjMYODuzZCdXdce1CNozceHr+a+OCMGQ'. The resource has been blocked."
And then the "KimaiWebLoader is not defined" I guess I need to rebuild the file but then how? Or should I just reinstall kimai from the beginning?

**Comment by kevinpapst** [comment, marked as answer]
Thankfully some clever engineers designed a feature called search:
https://github.com/kimai/kimai/discussions?discussions_q=is%3Aopen+integrity+digest

Looks promising: https://github.com/kimai/kimai/discussions/4124

**Comment by ilfungo** [comment]
The string I put in the search didn't give me any result. Thanks for your time.
